src/command/cmd_funcs.c

```diff
@@ -9296,33 +9296,74 @@ cmd_mam(ProfWin* window, const char* const command, gchar** args)
 gboolean
 cmd_editor(ProfWin* window, const char* const command, gchar** args)
 {
-    const char* filename = "/tmp/profanity.txt";
+    /* Build temp file name */
+    GString * tempfile = g_string_new(g_get_tmp_dir());
```

Comment:
Instead of creating a Gstring and appending several times. I would rather use `g_strdup_printf()`. It's just one call instead of plenty of lines :) You can format the string right there. And no need for `const char* filename = tempfile->str;` then too since it already returns a gchar.

Reply:
done in 8be46422d2ca5afff675ef3f7a2117c097a642a7